packages/dds/tree/src/shared-tree-core/editManagerSummarizer.ts

```diff
@@ -78,7 +86,11 @@ export class EditManagerSummarizer<TChangeset> implements Summarizable {
 	}
 
 	private summarizeCore(stringify: SummaryElementStringifier): ISummaryTreeWithStats {
-		const jsonCompatible = this.codec.encode(this.editManager.getSummaryData());
+		const context: EditManagerEncodingContext =
+			this.schema !== undefined
+				? { schema: { policy: defaultSchemaPolicy, schema: this.schema } }
```

Comment:
should we pass down a policy? is it okay to use the default here?

Reply:
Updated to pass down policy from `SharedTree` level